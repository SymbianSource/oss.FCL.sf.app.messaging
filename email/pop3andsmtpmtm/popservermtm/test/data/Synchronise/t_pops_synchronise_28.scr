+OK demon POP3 server ready (demon.co.uk)
+OK 
+OK 
+OK 11 16276
+OK 11 messages
1 1403
2 1403
3 1443
4 1437
5 1485
6 1597
7 1518
8 1560
9 1518
10 1354
11 1558
.
+OK 11 messages
1 t_pops_synchronise:message015
2 t_pops_synchronise:message016
3 t_pops_synchronise:message018
4 t_pops_synchronise:message019
5 t_pops_synchronise:message017
6 t_pops_synchronise:message023
7 t_pops_synchronise:message022
8 t_pops_synchronise:message024
9 t_pops_synchronise:message021
10 t_pops_synchronise:message020
11 t_pops_synchronise:message025
.
-ERR Unknown Command "CAPA"
+OK TOP 11 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message025; Tue, 14 Aug 2001 14:57:47 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id aa1110552; 14 Aug 2001 14:56 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e2ccad0@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 15:50:13 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHACF>; Tue, 14 Aug 2001 15:46:43 +0100
Message-ID: <t_pops_synchronise_message025@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 25
Date: Tue, 14 Aug 2001 15:38:55 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Twenty-Five Twenty-Five Twenty-Five Twenty-Five Twenty-Five Twenty-Five
.
+OK TOP 11 1 accepted
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message020; Tue, 14 Aug 2001 14:49:59 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id aa1114851; 14 Aug 2001 14:42 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e2498ab@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 15:41:16 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHABX>; Tue, 14 Aug 2001 15:43:09 +0100
Message-ID: <t_pops_synchronise_message020@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 20
Date: Tue, 14 Aug 2001 15:35:20 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Twenty Twenty Twenty Twenty Twenty Twenty Twenty Twenty Twenty Twenty 
.
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message021 997800562:10:06905:25; Tue, 14 Aug 2001 14:49:22 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id ac1114851; 14 Aug 2001 14:42 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e256c18@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 15:42:10 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHABZ>; Tue, 14 Aug 2001 15:43:37 +0100
Message-ID: <t_pops_synchronise_message021@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 21
Date: Tue, 14 Aug 2001 15:35:48 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Twenty-One Twenty-One Twenty-One Twenty-One Twenty-One Twenty-One Twenty-One
.
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message024; Tue, 14 Aug 2001 14:46:21 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id aa1027733; 14 Aug 2001 14:44 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e2800ec@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 15:45:00 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHAC1>; Tue, 14 Aug 2001 15:46:19 +0100
Message-ID: <t_pops_synchronise_message024@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 24
Date: Tue, 14 Aug 2001 15:38:32 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Twenty-Four Twenty-Four Twenty-Four Twenty-Four Twenty-Four Twenty-Four
.
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message022; Tue, 14 Aug 2001 14:46:17 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id ab1114851; 14 Aug 2001 14:42 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e25fbb6@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 15:42:47 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHAB8>; Tue, 14 Aug 2001 15:44:08 +0100
Message-ID: <t_pops_synchronise_message022@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 22
Date: Tue, 14 Aug 2001 15:36:20 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Twenty-Two Twenty-Two Twenty-Two Twenty-Two Twenty-Two Twenty-Two Twenty-Two
.
+OK 
Return-Path: <Andrew.Fawcett@teleca.com>
Received: from punt-1.mail.demon.net by mailstore for freddy@yashar.demon.co.uk
          id t_pops_synchronise:message023; Tue, 14 Aug 2001 14:45:47 GMT
Received: from mail.teleca.com ([195.166.58.131]) by punt-1.mail.demon.net
           id ad1114851; 14 Aug 2001 14:42 GMT
Received: from teleca126.teleca.com (unverified) by teleca161.teleca.com
 (Content Technologies SMTPRS 4.1.5) with ESMTP id <Tac100102555e26530d@teleca161.teleca.com> for <freddy
@yashar.demon.co.uk>;
 Tue, 14 Aug 2001 15:43:10 +0100
Received: by teleca126 with Internet Mail Service (5.5.2650.21)
        id <Q44VHACA>; Tue, 14 Aug 2001 15:44:30 +0100
Message-ID: <t_pops_synchronise_message023@JANUS.teleca.com>
From: Andrew Fawcett <Andrew.Fawcett@teleca.com>
To: 'freddy' <freddy@yashar.demon.co.uk>
Subject: Message 23
Date: Tue, 14 Aug 2001 15:36:42 +0100
Return-Receipt-To: Andrew Fawcett <Andrew.Fawcett@teleca.com>
MIME-Version: 1.0
X-Mailer: Internet Mail Service (5.5.2650.21)
Content-Type: text/plain;
        charset="iso-8859-1"

Twenty-Three Twenty-Three Twenty-Three Twenty-Three Twenty-Three
.
** Read cancelled
+OK 
