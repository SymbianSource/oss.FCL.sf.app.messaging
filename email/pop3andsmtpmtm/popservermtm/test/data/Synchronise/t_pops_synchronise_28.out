STAT
LIST
UIDL
CAPA
TOP 11 1
TOP 10 1
TOP 9 1
TOP 8 1
TOP 7 1
TOP 6 1
QUIT
